client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -359,27 +361,28 @@ export const PatternLibrary = ( {
 					</PatternLibraryBody>
 				) }
 
-				{ isEnabled( 'readymade-templates/showcase' ) && isHomePage && <ReadymadeTemplates /> }
-
-				{ ! isEnabled( 'readymade-templates/showcase' ) && isHomePage && (
-					<PatternsCopyPasteInfo theme="dark" />
-				) }
-
 				{ isHomePage && (
-					<CategoryGallery
-						title={ pageLayoutsHeading }
-						description={ translate(
-							'Start even faster with ready-to-use pages and preassembled patterns. Then tweak the design until it’s just right.'
-						) }
-						categories={ categories?.filter( ( c ) => c.pagePatternCount ) }
-						patternTypeFilter={ PatternTypeFilter.PAGES }
+					<PatternsCopyPasteInfo
+						theme={ isEnabled( 'readymade-template/showcase' ) ? 'gray' : 'dark' }
```

Comment:
Typo in the flag name
```suggestion
						theme={ isEnabled( 'readymade-templates/showcase' ) ? 'gray' : 'dark' }
```